OIPA/requirements.txt

```diff
@@ -1,41 +1,68 @@
-git+https://github.com/zimmerman-zimmerman/djorm-ext-pgfulltext.git
-git+https://github.com/gabber7/django-supervisor.git
-django-nested-admin==2.2.6
-django-autocomplete-light==2.3.3
-requests==2.11.1
+argh==0.26.2
```

Comment:
Did you copy-paste your `pip list` output into the requirements file? 

This causes dependencies of dependencies to be added, which causes loose dependencies (and a maintenance hell :) ). 

I would prefer to change this to just adding Django silk to the requirements and if this requires any library updates, change their version numbers.